src/framework/global/types/translatablestring.h

```diff
@@ -0,0 +1,173 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#ifndef MU_FRAMEWORK_TRANSLATABLESTRING_H
+#define MU_FRAMEWORK_TRANSLATABLESTRING_H
+
+#include "translation.h"
+
+#include "types/string.h"
+
+#ifndef NO_QT_SUPPORT
+#include <QString>
+#endif
+
+namespace mu {
+//! Note: in order to make the string visible for `lupdate`,
+//! you must write TranslatableString(...) explicitly.
+struct TranslatableString {
```

Comment:
It's quite a long name for something that will appear in lots of places. How about calling the struct `Translatable` since it doesn't inherit from `String`?